src/state/optionsSlice.ts

```diff
@@ -8,6 +8,7 @@ export type ChartOptions = {
   validateTooltipEventTypes?: ReadonlyArray<TooltipEventType>;
   // Should this instead be a property of a graphical item? Do we want to mix items with different data types in one chart?
   tooltipPayloadSearcher: TooltipPayloadSearcher | undefined;
+  barCategoryGap: number | string;
```

Comment:
I know this is for parity, but should we eventually have this be specific to Bar?

Reply:
BarChart plus ComposedChart I think. Yep, eventually